package.json

```diff
@@ -35,11 +35,11 @@
     "scripts": {
         "clean": "rimraf lib *.jar *.log",
         "compile": "tsc",
-        "test": "node scripts/download-and-run-rc.js && mocha 'test/**/*.js'",
-        "test:unit": "mocha 'test/unit/**/*.js'",
-        "test:integration": "node scripts/download-and-run-rc.js && mocha 'test/integration/**/*.js'",
+        "test": "node scripts/download-rc.js && node scripts/test-runner.js alltests",
+        "test:unit": "node scripts/test-runner.js unit",
+        "test:integration": "node scripts/download-rc.js && node scripts/test-runner.js integration",
         "validate-user-code": "tsc --build test/user_code/tsconfig.json",
-        "precoverage": "node scripts/download-and-run-rc.js",
+        "precoverage": "node scripts/download-rc.js && node scripts/test-runner.js startrc",
         "coverage": "rimraf coverage && nyc node_modules/mocha/bin/_mocha 'test/**/*.js' -- --reporter-options mochaFile=report.xml --reporter mocha-junit-reporter",
```

Comment:
I do not want to include coverage in test-runner, since we will want to use `node node_modules/rimraf/..` instead of `rimraf`(since we used in this way for mocha and I feel it is a good thing) and similarly for nyc, and I am not sure if it will break this complex command: `nyc node_modules/mocha/bin/_mocha 'test/**/*.js' -- --reporter-options mochaFile=report.xml --reporter mocha-junit-reporter`

Reply:
Whatever, I changed my mind, adding coverage to test-runner.js. Otherwise killing remote controller is impossible after coverage run.